src/games/strategy/engine/data/properties/DoubleProperty.java

```diff
@@ -49,7 +49,7 @@ public void setValue(final Object value) throws ClassCastException {
       // todo (kg) remove at a later point
       throw new RuntimeException(
           "Double and Number properties are no longer stored as Strings. You should delete your option cache, located at "
-              + new File(GameRunner2.getUserRootFolder(), "optionCache").toString());
+              + new File(ClientFileSystemHelper.getUserRootFolder(), "optionCache").toString());
```

Comment:
Will see a number of this update, methods moved from GameRunner2 were: getUserRootFolder(), getMapsFolder().

GetFile() in GameRunner2 was moved to LoadGameUtil.